resources/web/wwi/editor.js

```diff
@@ -41,7 +41,7 @@ class Editor extends DialogWindow { // eslint-disable-line no-unused-vars
       saveShortcut = 'Ctrl-S';
     else // macOS
       saveShortcut = 'Cmd-S';
-    this.menu.innerHTML = '<input type="image" id="webotsEditorMenuImage" width="17px" src="' + DefaultUrl.getImageUrl('menu') + '">' +
+    this.menu.innerHTML = '<input type="image" id="webotsEditorMenuImage" width="17px" src="' + DefaultUrl.wwiUrl() + 'images/' + 'menu.png">' +
```

Comment:
Why not using getImageUrl here?

Reply:
I see, it doesn't return the correct string... But then, why don't we remove this getImageUrl method which doesn't seem to be very useful. Is it used in many places?

Reply:
It is used in two places.
I can change it and remove the `url(..)` part so that it can be used here as well.